lib/dynamoid/config.rb

```diff
@@ -26,6 +26,7 @@ module Config
     option :sync_retry_max_times, default: 60 # a bit over 2 minutes
     option :sync_retry_wait_seconds, default: 2
     option :convert_big_decimal, default: false
+    option :convert_date_to_string, default: false
```

Comment:
To support existed projects and data we may need to specify this format not globally but for specific model and field (as well as allow to set it globally)

Reply:
@andrykonchin wouldn't the default value of `false` make it backwards compatible with any existing implementation?

Reply:
@pboling It would, sure.
But you could want to keep old format for existing tables and new format for new ones. You couldn't use new format if already have data in old one in any table.

Reply:
Oh, I see.  Agreed, this has to be model-specific per field.